tests/test_continuous_profiler.py

```diff
@@ -0,0 +1,237 @@
+import threading
```

Comment:
If you decide to go with the new `profiler` folder in the sdk that I suggested above, it'd make sense to do the same with `test_continuous_profiler.py` and `test_profiler.py` in the tests.